maat-court-data-api/src/main/java/gov/uk/courtdata/model/LaaTransactionLogging.java

```diff
@@ -13,9 +13,9 @@
 @Value
 public class LaaTransactionLogging {
 
-    private UUID laaTransactionId;
-    private Integer maatId;
-    private String caseUrn;
+    UUID laaTransactionId;
```

Comment:
Just being pedantic here, we should restrict the access fields and they should only be accessible within a class, the default value is package-private unless there is a reason I'm not aware of. I'm fine either way.